core/common/jwt-core/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2022 Amadeus
```

Comment:
I think the copyrights need to be additive, not replaced

Reply:
The fact is that I removed the two preexistent `-lib` modules (`token-validation-lib` and `token-generation-lib`) and created `jwt-core`. For some reasons git detected this as a file rename, but, in fact, this is a completely different file.